kustomize/micro/allocation.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: ingressgateway
+spec:
+  template:
+    spec:
+      containers:
+        - name: istio-proxy
+          resources:
+            limits:
+              memory: 1250Mi
+            requests:
+              memory: 1250Mi
```

Comment:
Can we get a readme or something on the profiles? I thought micro meant small install (in terms of resources) but based on this I guess its micro in terms of number of components?

Reply:
Yes, it is on number of components. 
The allocation file is mostly to test that it works and provide an example - I'll adjust this.